Repository/AbstractPhpcrRepository.php

```diff
@@ -110,4 +112,12 @@ protected function unresolvePath($path)
      * @return ArrayResourceCollection
      */
     abstract protected function buildCollection(array $nodes);
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getVersions($path)
+    {
+        return new VersionList($path, [new CmfResource($path)]);
```

Comment:
ok, will do that ...
but why is there an abstact repository with a implementation, that we don't extend instead?